research/sentiment_analysis/data/dataset.py

```diff
@@ -0,0 +1,9 @@
+import data.imdb as imdb
```

Comment:
nit: add license header, and from __future__ imports like other py files

Reply:
Please address this comment.